python/src/com/jetbrains/python/console/PydevConsoleCommunication.java

```diff
@@ -589,13 +587,19 @@ public XSourcePosition getSourcePositionForType(String type) {
    * Request that pydevconsole connect (with pydevd) to the specified port
    *
    * @param localPort port for pydevd to connect to.
+   * @param dbgOpts   additional debugger options (that are normally passed via command line) to apply
+   * @param extraEnvs
    * @throws Exception if connection fails
    */
-  public void connectToDebugger(int localPort) throws Exception {
+  public void connectToDebugger(int localPort, @NotNull Map<String, Boolean> dbgOpts, @NotNull Map<String, String> extraEnvs)
+    throws Exception {
     if (waitingForInput) {
       throw new Exception("Can't connect debugger now, waiting for input");
     }
-    Object result = myClient.execute(CONNECT_TO_DEBUGGER, new Object[]{localPort});
+    /* argument needs to be hashtable type for compatability with the RPC library */
+    Hashtable<String, Object> opts = new Hashtable<>(dbgOpts);
```

Comment:
You have to use it. Otherwise the RPC library wont be able to marshall the type. You need to upgrade the library if you want to use hashmap.